demosplan/DemosPlanCoreBundle/Addon/FrontendAssetProvider.php

```diff
@@ -39,10 +40,14 @@ public function getFrontendClassesForHook(string $hookName): array
             $hookData = $uiData['hooks'][$hookName];
             $manifestPath = DemosPlanPath::getRootPath($addonInfo->getInstallPath()).'/'.$uiData['manifest'];
 
+            // Return if no access granted for that addon at that entrypoint
+            if (array_key_exists('permissions', $hookData['options']) && 0 !== count(array_filter($hookData['options']['permissions'], fn (array $permission) => $this->permissions->hasPermission[$permission]))) {
+                return [];
+            }
```

Comment:
```suggestion
            if (!array_key_exists('permissions', $hookData['options']) ||
                !$this->permissions->hasPermissions($hookData['options']['permissions'])) {
                return [];
            }
```
if you want something likje 
```
  ui:
    manifest: dist/assets-manifest.json
    hooks:
      tag.extend.form:
        entry: TopicalTagLoader
        options:
          permissions:
            - 'feature_a'
            - 'feature_b'
          title: "Set Tag as Ai-Tag"
      tag.create.form:
        entry: TopicalTagToggle
        options:
          title: "Set Tag as Ai-Tag"
```
in the demosplan-addon.yml